tests/keras/backend/reference_operations.py

```diff
@@ -526,6 +528,101 @@ def resize_volumes(x, depth_factor, height_factor, width_factor, data_format):
     return x
 
 
+def ndim(x):
```

Comment:
I think that numpy arrays have a ndim attribute that you could use.

Reply:
sure . replaced by return x.ndim